ci/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM alpine:3.8 as conjur-test
+
+ARG KUBECTL_VERSION=1.17.0
+ARG KIND_VERSION=0.7.0
+
+RUN apk add --no-cache \
+    bash \
+    curl \
+    docker \
+    openssl \
+    wget
+
+# Install kubectl
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v${KUBECTL_VERSION}/bin/linux/amd64/kubectl && \
+    chmod +x ./kubectl && \
+    mv ./kubectl /usr/local/bin/kubectl
+
+# Install Kubernetes in Docker (kind)
+RUN curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v${KIND_VERSION}/kind-linux-amd64 && \
+    chmod +x ./kind && \
+    mv ./kind /usr/local/bin/kind
+
+# Install helm
+RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && \
+    chmod 700 get_helm.sh && \
+    ./get_helm.sh
+
+#COPY conjur-oss /root/conjur-oss/
```

Comment:
Needs removal?

Reply:
Yes. But now the entire file has been deleted.